Necromancer and illusionist spells

JamesMcClung opened the pull request:
Add special spells exclusive to necromancer and illusionist.

Also remove a few spells from the wizard spell list, and reword some existing spells.

JamesMcClung committed:
spells: wizards lose ...

green-flame blade
shadow blade
booming blade
fireball
witch bolt

JamesMcClung committed:
spells: reword some illusion spells...

disguise self
seeming
mirror image
hallucinatory terrain
symbol
glyph of warding
phantasmal force
major image
minor illusion
silent image
programmed illusion
project image

JamesMcClung committed:
spells: reword summon undead

JamesMcClung committed:
spells: add illusionist spells

JamesMcClung committed:
spells: add necromancer spells

bwhetherington commented on spells.yml:
```diff
@@ -3406,6 +3665,23 @@ fabricate:
+  components: S
+  concentration: false
+  description: |-
+    Make a melee spell attack against a creature within 5 feet of you that's the same size as you. On a hit, you become disguised as the target as if by _disguise self_ and vice versa.
```
Perhaps it'd be worthwhile to add a clause about how this can also be performed on a willing target?

bwhetherington commented on spells.yml:
```diff
@@ -6408,14 +6817,15 @@ minor_illusion:
     If you create an image of an object—such as a chair, muddy footprints, or a small chest—it must be no larger than a 5-foot cube. The image can't create sound, light, smell, or any other sensory effect. Physical interaction with the image reveals it to be an illusion, because things can pass through it.
 
-    If a creature uses a standard action to examine the sound or image, the creature can determine that it is an illusion with a successful Intelligence (Investigation) check against your spell save DC. If a creature discerns the illusion for what it is, the illusion becomes faint to the creature.
+    A creature can use the Study action to inspect the sound or image, making an Investigation check against your spell save DC. On a success, the creature discovers that the sound or image is illusory. If a creature discerns the illusion for what it is, the illusion becomes faint to that creature.
```
While we're modifying this, should we just use exactly the same language as with _major image_? I.e.:

> If a creature discerns the illusion for what it is, the creature can see through the image, and its other sensory qualities become faint to the creature.

I'm not sure what the right wording should be, but it seems odd that an Investigation check would let you see through _major image_ but not _minor illusion_.

JamesMcClung replied:
I ran into that when rewording them, and the catch is that minor illusion might just make a sound with nothing visible. I'll give it a shot though, because it should be clear that you can see through either on an Investigation

bwhetherington commented on spells.yml:
```diff
@@ -6453,11 +6863,9 @@ mirror_image:
+    Three illusory duplicates of yourself appear in your space. Until the spell ends, the duplicates move with you and mimic your actions, shifting position so it's impossible to track which image is real. You can use a standard Magic action to dismiss the illusory duplicates.
 
-    If you have three duplicates, you must roll a 6 or higher to change the attack's target to a duplicate. With two duplicates, you must roll an 8 or higher. With one duplicate, you must roll an 11 or higher.
+    Each time a creature targets you with an attack during the spell's duration, randomly determine whether the attack instead targets one of your duplicates. You and each duplicate have equal chances of being hit. For example, if you have three duplicates, there is a 25% chance the attack targets you and a 75% chance that it targets one of your duplicates.
```
I'm not sure I'm a fan of outright putting percentages here. It feels antithetical to D&D to not frame it in terms of dice, even if this does make more sense mathematically.

bwhetherington commented on spells.yml:
```diff
@@ -6885,6 +7313,28 @@ phantom_steed:
+  components: V
+  concentration: true
+  description: |-
+    Up to four illusory, Medium- or Small-sized phantom warriors appear in unoccupied spaces within range. Each creature's appearance and behavior is up to you. They appear real, visually and audibly, but are incorporeal.
```
Is it actually correct to leave "-" attached to Medium here?

JamesMcClung replied:
It is

bwhetherington commented on spells.yml:
```diff
@@ -9778,6 +10341,80 @@ tsunami:
+
+    These changes last until the spell ends or until you change them again. You can be as specific or as general as you want when making these changes; the spell automatically fills in missing details to match your proscribed reality.
+
+    If the spheres of two or more castings of _unreality_ would overlap, each point in space is controlled by the sphere whose center is closest to that point.
```
Minor question, but what would happen to a point equidistant from the centers of the spheres?

JamesMcClung replied:
I wish I could just say "partition space like a Voronoi diagram" but from but the intent is that those equidistant edges don't z-fight, they take the appearance dictated by the side you're on

JamesMcClung commented on spells.yml:
```diff
@@ -2442,6 +2590,21 @@ detect_thoughts:
+  casting_time: 1 standard action
+  components: V, M
+  concentration: true
+  description: Choose a 100-foot cube within range. For the duration, bright light in the cube becomes dim light light, and creatures in the cube take half damage from radiant damage.
```
The intent was just bright->dim. I could add some fluff saying how it suppresses harsh light in the area, which was what I was going for

JamesMcClung committed:
spells: more rewording ...

dim: add fluff
face swap: autohit willing
major image: "translucent"
minor illusion: "translucent"
mirror image: use d12

JamesMcClung commented on spells.yml:
```diff
@@ -1000,6 +1031,26 @@ burning_hands:
     - magus
     - sorcerer
     - wizard
+cadaverous_puppet:
+  casting_time: 1 standard action
+  components: V, S
+  concentration: true
+  description: |-
+    You wrest control of a Medium or smaller corpse or skeleton within range. You control the body for the duration of the spell.
+
+    When you cast this spell, and as a minor action on subsequent turns, you can cause the body to stand up if it is prone and walk up to 20 feet, where it remains standing unless you will otherwise. You can also have the body perform simple tasks as part of this action, such as opening a door or pushing a rock. You can't cause the body to attack in this way, however.
+
+    The body counts as an object. It has 15 hit points, 10 AC, and immunity to necrotic, poison, and psychic damage. It uses the Strength, Dexterity, and Constitution scores it had in life. If the body is reduced to 0 hit points, it is destroyed and the spell ends.
+  duration: Up to 8 hours
+  higher_level: When you cast this spell using a spell slot of 3rd level, you can animate a Large or smaller body. When you cast this spell using a spell slot of 4th level, you can animate a Huge or smaller body. When you cast this spell using a spell slot of 5th level or higher, you can animate a body of any size. The body also has an additional 15 hp for each slot level above 2nd.
+  level: 2
+  name: Cadaverous Puppet
+  range: 60 feet
+  sources:
+    - 2D
+  tags:
+    - arcane
+    - necromancy
```
So this spell needs a a buff. My current thinking is:
- Add a minor action cantrip that makes a body stand up, walk 20 feet, and stay standing until the end of the turn. Cantrip means it can be paired with e.g. _bloat burst_ on the same turn, but an 8 hour duration would be a bit much I think.
- Keep this spell with its 8 hour duration + concentration, but make the casting and reuse a free action. This way, it doesn't use your very limited AP, so it's more effective long term in combat. Also, it should let you swap targets, so if your body dies you can replace it without burning another spell slot. 8 hours means you can still use it for fun stuff out of combat like scouting with _eyes of the dead_.

bwhetherington replied:
I think I'd be fine with those changes.

JamesMcClung committed:
spells: buff cadaverous puppet

JamesMcClung committed:
spells: add deadwalk

JamesMcClung committed:
spells: fix typos

bwhetherington reviewed (approved):
All in all, looks good to me.
